src/electionguard/big_integer.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Tuple, Union
+from base64 import b16decode
+
+# pylint: disable=no-name-in-module
+from gmpy2 import mpz
+
+
+def _hex_to_int(input: str) -> int:
+    """Given a hex string representing bytes, returns an int."""
+    return int(input, 16)
+
+
+def _int_to_hex(input: int) -> str:
+    """Given an int, returns a hex string representing bytes."""
+    hex = format(input, "02X")
+    if len(hex) % 2:
```

Comment:
Why pad numbers to an odd length?  I see this was happening previously, but seems like this might warrant a comment of explanation for why this is happening, and maybe method whose name documents what's happening like `hex = pad_to_even_length(hex)`, and/or maybe a dedicated unit test or two.

Reply:
I will make a note here.

Reply:
It is actually padding if it IS an odd length to make sure that it has an even length in the end.